BriefAssistant/Models/Address.cs

```diff
@@ -10,108 +10,108 @@ namespace BriefAssistant.Models
-        IN,
-        [EnumMember]
-        KS,
-        [EnumMember]
-        KY,
-        [EnumMember]
-        LA,
-        [EnumMember]
-        MA,
-        [EnumMember]
-        MD,
-        [EnumMember]
-        ME,
-        [EnumMember]
-        MI,
-        [EnumMember]
-        MN,
-        [EnumMember]
-        MO,
-        [EnumMember]
-        MS,
-        [EnumMember]
-        MT,
-        [EnumMember]
-        NC,
-        [EnumMember]
-        ND,
-        [EnumMember]
-        NE,
-        [EnumMember]
-        NH,
-        [EnumMember]
-        NJ,
-        [EnumMember]
-        NM,
-        [EnumMember]
-        NV,
-        [EnumMember]
-        NY,
-        [EnumMember]
-        OH,
-        [EnumMember]
-        OK,
-        [EnumMember]
-        OR,
-        [EnumMember]
-        PA,
-        [EnumMember]
-        RI,
-        [EnumMember]
-        SC,
-        [EnumMember]
-        SD,
-        [EnumMember]
-        TN,
-        [EnumMember]
-        TX,
-        [EnumMember]
-        UT,
-        [EnumMember]
-        VA,
-        [EnumMember]
-        VT,
-        [EnumMember]
-        WA,
-        [EnumMember]
-        WI,
-        [EnumMember]
-        WV,
-        [EnumMember]
-        WY
+        [EnumMember(Value = "AL")]
+        Alabama,
+        [EnumMember(Value = "AK")]
+        Alaska,
+        [EnumMember(Value = "AZ")]
+        Arizona,
+        [EnumMember(Value = "AR")]
+        Arkansas,
+        [EnumMember(Value = "CA")]
+        California,
+        [EnumMember(Value = "CO")]
+        Colorado,
+        [EnumMember(Value = "CT")]
+        Connecticut,
+        [EnumMember(Value = "DE")]
+        Delaware,
+        [EnumMember(Value = "DC")]
+        DistrictOfColumbia,
+        [EnumMember(Value = "FL")]
+        Florida,
+        [EnumMember(Value = "GA")]
+        Georgia,
+        [EnumMember(Value = "HI")]
+        Hawaii,
+        [EnumMember(Value = "ID")]
+        Idaho,
+        [EnumMember(Value = "IL")]
+        Illinois,
+        [EnumMember(Value = "IN")]
+        Indiana,
+        [EnumMember(Value = "IA")]
+        Iowa,
+        [EnumMember(Value = "KS")]
+        Kansas,
+        [EnumMember(Value = "KY")]
+        Kentucky,
+        [EnumMember(Value = "LA")]
+        Louisiana,
+        [EnumMember(Value = "ME")]
+        Maine,
+        [EnumMember(Value = "MD")]
+        Maryland,
+        [EnumMember(Value = "MA")]
+        Massachusetts,
+        [EnumMember(Value = "MI")]
+        Michigan,
+        [EnumMember(Value = "MN")]
+        Minnesota,
+        [EnumMember(Value = "MS")]
+        Mississippi,
+        [EnumMember(Value = "MO")]
+        Missouri,
+        [EnumMember(Value = "MT")]
+        Montana,
+        [EnumMember(Value = "NE")]
+        Nebraska,
+        [EnumMember(Value = "NV")]
+        Nevada,
+        [EnumMember(Value = "NH")]
+        NewHampshire,
+        [EnumMember(Value = "NJ")]
+        NewJersey,
+        [EnumMember(Value = "NewMexico")]
```

Comment:
Should this be NM instead of NewMexico? It doesn't really match any of the others.